controllers/amazoncloudwatchagent_controller.go

```diff
@@ -40,6 +43,80 @@ type Params struct {
 	Config   config.Config
 }
 
+func (r *AmazonCloudWatchAgentReconciler) findCloudWatchAgentOwnedObjects(ctx context.Context, owner v1alpha1.AmazonCloudWatchAgent) (map[types.UID]client.Object, error) {
+	// Define a map to store the owned objects
+	ownedObjects := make(map[types.UID]client.Object)
+	selector := manifestutils.SelectorLabels(owner.ObjectMeta, "*")
```

Comment:
nit: Change the "*" to an empty string to make it clear that we're not using it. Without knowing what this function does, I would assume "*" selects everything.